site/en/android/adb-over-wifi.md

```diff
@@ -0,0 +1,83 @@
+
+1.  Ensure that your development machine and Android device are on the same wireless network.
+1.  Ensure that your network allows traffic on port `5555` (or another manually chosen port). Some enterprise and home networks have firewalls that will prevent ADB from functioning. You can also use a mobile phone as a wireless network hub to provide a network that allows port `5555` traffic, although this may result in a large amount of network traffic and battery drain for the phone.
+1.  For non-ChromeOS phones and tablets, you need to enable TCP/IP mode. To do so, connect your Android device to your development machine via USB and run the command `adb tcpip 5555` on the development machine. This will put the device into TCP/IP mode listening on port 5555. Once enabled, you can disconnect the USB cable.
+1.  Determine the IP address of your device. [IPv4 addresses](https://en.wikipedia.org/wiki/Internet_Protocol_version_4) are the most common type of addresses and look something like this: `192.168.1.4`. [IPv6 addresses](https://en.wikipedia.org/wiki/IPv6_address) are becoming more prevalent and look something like this: `2001:0db8:3c4d:0015:0000:0000:1a2f:1a2b`. If your device lists both, use the IPv4 address.
+    1.  On phones/tablets, go to Android settings, information, and scroll down until you see your **IP address**.
+    1.  On ChromeOS, click the clock at the bottom right, click the drop-down under the network icon, and then click on the network you are currently connected to. Under the **Network** heading you will see the IP address assigned to your Chromebook.
+1.  On your development machine, type `adb connect IP_ADDRESS_FROM_STEP4`, e.g., `adb connect 192.168.1.4`. You can also manually specify the port like `adb connect 192.168.1.4:5555`.
+1.  Use ADB or Android Studio as usual. You can verify the connection by running `adb devices`.
+
+!!! aside.message--tip
+**Tip:** For more information, see [Connect to a device over Wi-Fi](https://developer.android.com/tools/adb#wireless).
+!!!
+
+# ADB over Ethernet
+
+In some cases, it may not be possible to use a Wi-Fi network due to firewall restrictions, or a wired solution may simply be preferable. In these cases, you can connect over an Ethernet cable. The steps are similar to ADB over Wi-Fi but using an Ethernet cable.
+
+## Prerequisites
+
+Before you get started, you need to ensure that your device supports Ethernet connectivity:
+
+- **Ethernet cable:** A physical cable to run between your devices. Ethernet types Cat 5/5e/6/6a are all fine.
+- **Ethernet port:** An Ethernet port for both your test device and your development machine.
+
+!!! aside.message--note
+**Note:** If your device does not have an Ethernet port, you can instead use an Ethernet dongle that connects to your device via USB and provides Ethernet capability.
+!!!
+
+![A USB-A Ethernet dongle.](ix://android/adb/ethernet.jpg)
+_A USB-A Ethernet dongle._
```

Comment:
```suggestion
#[A USB-A Ethernet dongle.](ix://android/adb/ethernet.jpg)
```

We actually have a dedicated figure element specifically for this kind of setup. Replacing the `!` with a `#` should do it.